README.md

```diff
@@ -68,18 +67,18 @@
 
 ### Key Features <a name="key-features"></a>
 
-- **Users can see an updated list of SpaceX rockets and missions**
-- **Users can join or leave any mission**
-- **Users can reserve or cancel any rocket**
-- **Users can see reserved rockets and joined missions in the My Profile section**
+- **Users can filter stocks for specific market.**
+- **Users can filter stocks using name or symbol.**
+- **Users can see detailed information about any stock.**
+- **The application is fully responsive.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo](https://space-travelers-hub-27nq.onrender.com/)
+- [Live Demo](https://stock-viewer.onrender.com/)
```

Comment:
- Kindly, you forget to add the video presentation :+1: , please add it either in your pr description or in the readme as it is part of the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)
![Screenshot from 2023-03-17 00-08-11](https://user-images.githubusercontent.com/52010789/225753168-0d37c10a-a80c-4810-b135-20e5f3180527.png)

Reply:
My bad, I just updated the PR with the video, thank you 👌